# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysvinit/sysvinit.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Rediffed for sysvinit-2.86, altugh it is a question why we actually patch it
away ...

  - Rene Rebe <rene@exactcode.de>

--- sysvinit-2.86/src/Makefile	2004-06-09 14:47:45.000000000 +0200
+++ sysvinit-2.86-fixed/src/Makefile	2004-12-11 18:18:10.228142800 +0100
@@ -24,11 +24,11 @@
 MAN8	+= shutdown.8 telinit.8
 
 ifeq ($(DISTRO),)
-BIN	+= mountpoint
-SBIN	+= sulogin bootlogd
-USRBIN	+= utmpdump wall
-MAN1	+= mountpoint.1 wall.1
-MAN8	+= sulogin.8 bootlogd.8
+BIN	+= 
+SBIN	+= sulogin
+USRBIN	+= wall
+MAN1	+= wall.1
+MAN8	+= sulogin.8
 endif
 
 ifeq ($(DISTRO),Debian)
@@ -71,9 +71,6 @@
 mountpoint:	mountpoint.o
 		$(CC) $(LDFLAGS) -o $@ mountpoint.o
 
-utmpdump:	utmpdump.o
-		$(CC) $(LDFLAGS) -o $@ utmpdump.o
-
 runlevel:	runlevel.o
 		$(CC) $(LDFLAGS) -o $@ runlevel.o
 
